code/wdb/datastripe.q

```diff
@@ -60,25 +58,30 @@ if[.ds.datastripe;.proc.addinitlist[(`initdatastripe;`)]];
 \d .ds
 
 upserttopartition:{[dir;tablename;keycol;enumdata;nextp]
-        /- function takes a (dir)ectory handle, tablename as a symbol
-        /- column to key table on, an enumerated table and a timestamp.
-        /- partitions the data on the keycol and upserts it to the given dir
-        /- get unique sym from table
-        s:first raze value'[?[enumdata;();1b;enlist[keycol]!enlist keycol]];
-        /- get process specific taildir location
-        dir:` sv dir,.proc.procname,`$ string .wdb.currentpartition;
-        /- get symbol enumeration
-        partitionint:`$string (where s=value [`.]keycol)0;
-        /- create directory location for selected partition
-        directory:` sv .Q.par[dir;partitionint;tablename],`;
-        .lg.o[`save;"Saving ",string[s]," data from ",string[tablename]," table to partition ",string[partitionint],". Table contains ",string[count enumdata]," rows."];
-        .lg.o[`save;"Saving data down to ",string[directory]];
-        /- upsert select data matched on partition to specific directory
-        .[upsert;
-          (directory;enumdata);
-          {[e] .lg.e[`upserttopartition;"Failed to save table to disk : ",e];'e}
-        ];
-        };
+    /- function takes a (dir)ectory handle, tablename as a symbol
+    /- column to key table on, an enumerated table and a timestamp.
+    /- partitions the data on the keycol and upserts it to the given dir
+    
+    /- get unique sym from table
+    s:first raze value'[?[enumdata;();1b;enlist[keycol]!enlist keycol]];
+
+    /- get process specific taildir location
+    dir:` sv dir,.proc.procname,`$ string .wdb.currentpartition;
+
+    /- get symbol enumeration
+    partitionint:`$string (where s=value [`.]keycol)0;
+
+    /- create directory location for selected partition
+    directory:` sv .Q.par[dir;partitionint;tablename],`;
+    .lg.o[`save;"Saving ",string[s]," data from ",string[tablename]," table to partition ",string[partitionint],". Table contains ",string[count enumdata]," rows."];
+    .lg.o[`save;"Saving data down to ",string[directory]];
+
+    /- upsert select data matched on partition to specific directory
+    .[upsert;
+        (directory;enumdata);
+        {[e] .lg.e[`upserttopartition;"Failed to save table to disk : ",e];'e}
+    ];
+    };
```

Comment:
with the indentation change, it's difficult to see what if anything actually changed here?

Reply:
no changes made here, just changing indentation and spacing to try and improve readability

Reply:
Ok, I'll keep this in mind going forward